### ISSUE TITLE
Hosts file not being updated after container stop/start

### ISSUE THREAD

**Issue by MikeGlue** [Microsoft/navcontainerhelper#48]
Not sure when this issue started as I know this functionality worked at some point, but it doesn't appear as if the hosts file is being updated when I stop and start the container.

I'm creating the container like this:

$imageName = "navdocker.azurecr.io/dynamics-nav:11.0.19494.0-finus"

New-NavContainer -containerName "MyContainer" `
                 -accept_eula `
                 -accept_outdated `
                 -auth "NavUserPassword" `
                 -Credential $credential `
                 -doNotExportObjectsToText `
                 -imageName $imageName `
                 -includeCSide `
                 -licenseFile $licenseFile `
                 -memoryLimit 3G `
                 -updateHosts `
                 -useSSL `
                 -additionalParameters @("--restart no") `
                 -alwaysPull `
                 -shortcuts Desktop

**Comment by freddydk** [comment]
I had this happen once, way back. Edited the hosts file manually and then it started updating again.
Probably should revisit the update code to see if it somehow can come into a situation like this

**Comment by freddydk** [comment]
Mike - this might be related to the other scenario where the services didn't start.
The hosts file is updated by the scripts as well and if they don't run or if they fail, this would be a symptom.
I would like to have a look at the container where this doesn't work.
thanks

**Comment by MikeGlue** [comment]
Ahh, I meant to come back to this. I went through this looking for a cause, but I found that the hosts file was updating, but it took upwards of 2-3 minutes to get updated. The scenario that was initially done was start the container, and when the docker start command "finished", jump into one of the clients. This apparently is not enough time to wait. Unfortunately I've removed the troublesome container already and as far as the hosts file goes, as long as I wait a few minutes it seems to get updated just fine now. I don't see this much of a delay when using Server 2016 so I am chalking it up to another "difference" with Windows 10.
